package.json

```diff
@@ -7,29 +7,26 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
-
     "bootstrap": "^4.6.0",
-
     "moment": "^2.29.1",
     "react": "^17.0.2",
     "react-bootstrap": "^1.6.1",
     "react-datepicker": "^4.1.1",
     "react-dom": "^17.0.2",
     "react-icons": "^4.2.0",
-
     "react-p5": "^1.3.19",
-
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.3",
     "web-vitals": "^1.1.2"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "node backend/server.js",
```

Comment:
I don't think we want this change.... we should be using the heroku backend, right? @allinthemind66